pdf/model/image.go

```diff
@@ -255,28 +254,37 @@ func (ih DefaultImageHandler) NewImageFromGoImage(goimg goimage.Image) (*Image,
 		draw.Draw(m, m.Bounds(), goimg, b.Min, draw.Src)
```

Comment:
Might be good to update the `b` variable as it is used below for copying the image data.

```
draw.Draw(m, m.Bounds(), goimg, b.Min, draw.Src)
b = m.Bounds()
```

Reply:
Done, simplified it slightly